### PR TITLE
Correcting function name, clarifying a bit

### PR DESCRIPTION
**Pull request by MichaelCrimando**
This pull request [fixes existing content]

## Summary of Changes
- The documentation for ObjC creates a "start" function, but the code in the AppDelegate references a "connect" function. Changed the ObjC code to just make a "connect" function (matching the same code in Swift)
- The documentation skips creating a "connect" function until the very end. When it should be created in step 9

### CODE REVIEW

**Comment by MichaelCrimando** [comment]
@theresalech I'm trying to get this fixed before the Hackathon on 9/11. What do we have to do to get it merged?

File: docs/Getting Started/Integration Basics/index.md [hunk 1]
```diff
@@ -312,24 +312,29 @@ sdlManager = SDLManager(configuration: configuration, delegate: self)
 ```
 
 ### 9. Start the SDLManager
-The manager should be started as soon as possible in your application's lifecycle. We suggest doing this in the `didFinishLaunchingWithOptions()` method in your *AppDelegate* class. Once the manager has been initialized, it will immediately start watching for a connection with the remote system. The manager will passively search for a connection with a SDL Core during the entire lifespan of the app. If the manager detects a connection with a SDL Core, the `startWithReadyHandler` will be called.
+The manager should be started as soon as possible in your application's lifecycle. We suggest doing this in the `didFinishLaunchingWithOptions()` method in your *AppDelegate* class. Once the manager has been initialized, it will immediately start watching for a connection with the remote system. The manager will passively search for a connection with a SDL Core during the entire lifespan of the app. If the manager detects a connection with a SDL Core, the `startWithReadyHandler` will be called. Create a new function in the *ProxyManager* class called `connect`.
```
Comment (joeljfischer) [inline docs/Getting Started/Integration Basics/index.md]: Move the new sentence into its own paragraph.

**Comment by MichaelCrimando** [comment]
Doh, I was wondering why I never heard anything back on this. I must have missed the email. Looking int this again

**Comment by MichaelCrimando** [comment]
@joeljfischer  should be good to go now

**Comment by joeljfischer** [comment]
@MichaelCrimando Hi Michael, sorry to take so long on this. There were extensive changes in the pipeline for the documentation. Could you please check on your changes and update them for the new version of the documentation?

File: docs/Getting Started/Integration Basics/index.md [hunk 2]
```diff
@@ -312,24 +312,29 @@ sdlManager = SDLManager(configuration: configuration, delegate: self)
...
-    // Your app has successfully connected with the SDL Core.
-  }
-}];
+- (void)connect {
+    [self.sdlManager startWithReadyHandler:^(BOOL success, NSError * _Nullable error) {
+        if (success) {
+            // Your app has successfully connected with the SDL Core
+        }
+    }];
+}
```
Comment (MichaelCrimando) [inline docs/Getting Started/Integration Basics/index.md]: Still valid with new documentation

File: docs/Getting Started/Integration Basics/index.md (hunk 1 above)
Comment (MichaelCrimando) [inline docs/Getting Started/Integration Basics/index.md]: Still valid with new documentation

File: docs/Getting Started/Integration Basics/index.md [hunk 3]
```diff
@@ -412,7 +417,7 @@ static NSString* const AppId = @"<#App Id#>";
     return self;
 }
 
-- (void)start {
+- (void)connect {
```
Comment (MichaelCrimando) [inline docs/Getting Started/Integration Basics/index.md]: All still valid in latest documentation

**Comment by MichaelCrimando** [comment]
@joeljfischer All of the changes i made with this pull request are still valid and needed. Can we just PR it or is there anything else we need to do ?

**Comment by joeljfischer** [comment]
Nope it can happen in this PR if you want to merge it back into your branch, or make a new PR. Whichever you prefer.

**Comment by MichaelCrimando** [comment]
@joeljfischer Ok I believe I have everything setup correctly now through this PR

File: docs/Getting Started/Integration Basics/index.md [hunk 4]
```diff
@@ -340,22 +340,29 @@ sdlManager = SDLManager(configuration: configuration, delegate: self)
...
+    }];
+}
 ```
 
 ##### Swift
 ```swift
-sdlManager.start { (success, error) in
-    if success {
-      // Your app has successfully connected with the SDL Core.
+    func connect() {
```
Comment (joeljfischer) [inline docs/Getting Started/Integration Basics/index.md]: This method appears to be inset by about 4 characters

**Pull request merged**